guides/common/modules/proc_installing-convert2rhel.adoc

```diff
@@ -0,0 +1,29 @@
+[id="installing-convert2rhel_{context}"]
+= Installing Convert2RHEL
+
+The following procedure makes Convert2RHEL available to the hosts you are converting.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* *>* *Products* and click *Create Product*.
+. In the *Name* field, enter the name of the product.
+{Project} automatically completes the *Label* field based on your entry for *Name*.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. You do not need to complete *SSL Client Cert* or *SSL Client Key*.
+. Optional: From the *Sync Plan* list, select an existing sync plan or click *Create Sync Plan* to create a sync plan for your product requirements.
+. In the *Description* field, enter a description of the product.
+. Click *Save*.
+. Click the *Repositories* tab.
+. To add a repository, click *New Repository*.
+. In the *Name* field, enter the name of the repository.
+. From the *Type* list, set the content type to Yum.
+. Do not restrict the architecture or the OS version.
+. Use https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/ as the upstream URL.
+. Disable the *Verify SSL* checkbox.
```

Comment:
This is a temporary step, until the SSL on this URL is fixed. We shouldn't recommend to ignore security issues but I don't have a date, when it will be fixed. We should review this at the time of the GA.

Reply:
Will recheck at GA.